docs/introduction/get-started/p2p-private-offchain-txs.md

```diff
@@ -97,4 +97,3 @@ You have successfully configured and used the Miden client to interact with a Mi
 You have performed basic Miden rollup operations like sending transactions, generating and consuming notes.
```

Comment:
About this sentence

```
You have performed basic Miden rollup operations like sending transactions, generating and consuming notes.
```

In my understanding, we don't really send transactions in Miden. This is the difference to Ethereum. Can you maybe reformulate this?